searx/settings.yml

```diff
@@ -339,6 +339,17 @@ engines:
     disabled : True
     shortcut : or
 
+  - name : pdbe
+    engine : pdbe
+    shortcut : pdb
+# Hide obsolete PDB entries.
```

Comment:
Some of the content on the site are not transmitted over `https` so you might have to disable it by default. One of the project's devs could probably chime in.